### ISSUE TITLE
Support for Firefox Extended Support Release

### ISSUE THREAD

**Issue by fmarier** [mozilla/persona#2271]
If we don't already test Persona on the [ESR versions](https://www.mozilla.org/en-US/firefox/organizations/) of Firefox on Desktop, we should :)

On the other hand, if we already do, then we should add it to the [browser compatibility table](https://developer.mozilla.org/en-US/docs/persona/Browser_compatibility).

**Comment by callahad** [comment]
As far as I know, we support: ESR, Oldstable, Stable, Beta, Aurora, and Nightly.

I changed the Firefox listing to "All Current Release Channels, plus Previous Stable Release" though that's still not completely explicit.

Any thoughts on how to phrase this so that it's complete _and_ easy to grasp at a glance? Or is the change I made sufficient? I really don't want to add another footnote. Maybe a new section labeled "Supported Firefox Release Channels?"
